Analysis/Tasks/PWGCF/FemtoDream/include/FemtoDream/FemtoDerived.h

```diff
@@ -24,62 +24,72 @@
 
 namespace o2::aod
 {
+
+/// FemtoDreamCollision
 namespace femtodreamcollision
 {
-DECLARE_SOA_COLUMN(MultV0M, multV0M, float);
-DECLARE_SOA_COLUMN(Sphericity, sphericity, float);
+DECLARE_SOA_COLUMN(MultV0M, multV0M, float);       ///< V0M multiplicity
```

Comment:
The syntax to be put automatically in our documentation for the comments is : //! COMMENT

Reply:
Also below...